### PR TITLE
chore: update vulnerable mongo driver

### PR DESCRIPTION
**Pull request by aaron-steinfeld**
## Description
Updating mongo dependency due to https://snyk.io/vuln/SNYK-JAVA-ORGMONGODB-1079241

### CODE REVIEW

**Review by GurtejSohi — APPROVED** [review]

**Comment by aaron-steinfeld** [comment]
Overriding merge to unblock since it has one approval and codeowners here are outdated. @kotharironak have there been any discussions about updating or expanding the codeowners for HT repos?

**Pull request merged**
